src/test/java/org/elasticsearch/test/discovery/ClusterDiscoveryConfiguration.java

```diff
@@ -61,24 +63,25 @@ public Settings transportClient() {
 
     public static class UnicastZen extends ClusterDiscoveryConfiguration {
 
-        private static final AtomicInteger portCounter = new AtomicInteger();
+        private static int nextPort = calcBasePort();
```

Comment:
can we add a comment that this the next port we will try to assign and that it is advanced with every attempt?